Dockerfiles/manifests/agent-only/README.md

```diff
@@ -1,11 +1,13 @@
 The kubernetes manifests found in this directory have been automatically generated
 from the [helm chart `datadog/datadog`](https://github.com/DataDog/helm-charts/tree/master/charts/datadog)
-version 2.19.8 with the following `values.yaml`:
+version 2.28.11 with the following `values.yaml`:
 
 ```yaml
 datadog:
   collectEvents: true
   leaderElection: true
+  apm:
+    socketEnabled: false
```

Comment:
so we want do disable APM/trace-agent completely ?

Reply:
I only wanted to have the same things in the generated manifests.
We used to not have `trace-agent` in the manifests generated from this `agent-only` `values.yaml` because it was disabled by default.
But the Helm chart has been updated and the `trace-agent` is now enabled by default in the Helm chart.
But as far as the statically generated manifests are concerned, I thought it makes more sense to still not have the `trace-agent` in the `agent-only` manifests and to have it only in the `all-containers` manifests like we used to have.